.circleci/config.yml

```diff
@@ -12,6 +12,7 @@ jobs:
           command: |
             apt-get update --yes
             apt-get install sudo git wget curl software-properties-common apt-transport-https --yes
+            sudo apt-get install default-jdk --yes
```

Comment:
The reason we need to install JDK is MLeap for R requires it, correct?

Reply:
@harupy Good point. I just realized that must have been the case.
Then I guess it makes more sense for this to be part of the PR for supporting mleap in R